manifests/istio-control/istio-discovery/templates/istiod-injector-configmap.yaml

```diff
@@ -6,6 +6,7 @@ metadata:
   namespace: {{ .Release.Namespace }}
   labels:
     release: {{ .Release.Name }}
+    istio.io/rev: {{ .Values.revision }}
```

Comment:
This should exist on master and I think will conflict as I have it as `{{ .Values.revision | default "default" }}`. Probably needs a rebase?